notebooks/md-table-tests-v1.ipynb

```diff
@@ -7,21 +7,17 @@
    "source": [
```

Comment:
<p>Modified markdown table. It should render </p><ul><li>as diff highlighted raw markdown text</li><li>with each row on new line</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/PythonDataScienceHandbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>